demisto_sdk/dev_tools/uploader.py

```diff
@@ -0,0 +1,49 @@
+from demisto_sdk.common.tools import print_color, LOG_COLORS, print_v
+import demisto_client
+import os
+
+
+class Uploader:
+    DEMISTO_API_KEY_ENV = 'DEMISTO_API_KEY'
+
+    def __init__(self, infile: str, url: str, insecure: bool = False, verbose: bool = False):
+        self.infile = infile
+        self.base_url = url
+        self.log_verbose = verbose
+        self.client = demisto_client.configure(base_url=self.base_url,
+                                               api_key=self.get_api_key(),
+                                               verify_ssl=not insecure)
+
+    def get_api_key(self):
+        """Retrieve the API Key
+
+        Raises:
+            RuntimeError: if the API Key environment variable is not found
+
+        Returns:
+            str: API Key
+        """
+        ans = os.environ.get(self.DEMISTO_API_KEY_ENV, None)
+        if ans is None:
+            raise RuntimeError(f'Error: Environment variable {self.DEMISTO_API_KEY_ENV} not found')
+
+        return ans
+
+    def upload(self):
```

Comment:
what do you think about adding a --unify flag which will upload a package for you

Reply:
Possible, but should be more complicated - we won't just add integrations but a lot of directories and files, not sure if demisto_client supports these operations

Reply:
ok, lets open an issue for this enhancement please